.travis.yml

```diff
@@ -40,10 +42,15 @@ before_install:
   - eval `opam config env`
   # Install odoc & ocp-indent
   - opam install odoc ocp-indent.1.8.1
+  # Install sphinx libs
+  - pip install --user sphinx
+  - pip install --user recommonmark
+  - pip install --user sphinx_rtd_theme
+  - pip install --user sphinx-markdown-tables
```

Comment:
Is there any way to add these as package deps in the `addons` section (which is cleaner) ? I.e. do these packages exist as ubuntu-packages or alternatively, does some additional ppa allows to use them, in the same vein as the `avsm` ppa adds versioned opam packages.